internal/iocopy/iocopy_test.go

```diff
@@ -0,0 +1,136 @@
+package iocopy_test
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/kopia/kopia/internal/iocopy"
+)
+
+const (
+	testBuf    = "Hello, World!"
+	lenTestBuf = len(testBuf)
+)
+
+type errorWriter struct{}
+
+func (errorWriter) Write(p []byte) (n int, err error) {
+	return 0, errors.New("write error") //nolint:err113
+}
+
+func TestGetBuffer(t *testing.T) {
+	buf := iocopy.GetBuffer()
+	if len(buf) != iocopy.BufSize {
+		t.Errorf("Buffer size is not equal to BufSize, got %d", len(buf))
+	}
+}
+
+func TestReleaseBuffer(t *testing.T) {
+	buf := iocopy.GetBuffer()
+	iocopy.ReleaseBuffer(buf)
+	buf2 := iocopy.GetBuffer()
+
+	if &buf[0] != &buf2[0] {
+		t.Errorf("Buffer was not recycled after ReleaseBuffer")
+	}
+}
+
+func TestCopy(t *testing.T) {
+	src := strings.NewReader(testBuf)
+	dst := &bytes.Buffer{}
+
+	n, err := iocopy.Copy(dst, src)
+	if err != nil {
```

Comment:
please switch to `require` for assertions, otherwise this is great, thank you so much!

Reply:
Done.